easybuild/easyconfigs/a/AMD-SecureRNG/AMD-SecureRNG-2.2-GCC-9.3.0.eb

```diff
@@ -0,0 +1,23 @@
+easyblock = 'Binary'
+
+name = 'AMD-SecureRNG'
+version = '2.2'
+
+homepage = 'https://developer.amd.com/amd-cpu-libraries/rng-library/'
+description = """The AMD Secure Random Number Generator (RNG) is a library that provides APIs to access the
+cryptographically secure random numbers generated by AMD’s hardware-based random number generator implementation."""
+
+toolchain = {'name': 'GCC', 'version': '9.3.0'}
+
+source_urls = ['http://developer.amd.com/wordpress/media/files/']
+sources = ['aocl-securerng-linux-gcc-%(version)s-4.tar.gz']
+checksums = ['27920021ddb7e3ace006747c838d5330fefb863d62dc7ed98a81880f2a9ef66d']
+
+extract_sources = True
+
+sanity_check_paths = {
+    'files': [],
+    'dirs': ['lib'],
```

Comment:
This could be a bit more specific, a sanity check like this isn't very useful imho :)

Reply:
fixed in #11935